src/flake8/plugins/reporter.py

```diff
@@ -7,6 +7,7 @@
 from flake8.plugins.finder import LoadedPlugin
 
 LOG = logging.getLogger(__name__)
+DEFAULT_REPORTER = "default"
```

Comment:
seems unrelated and an unhelpful "constant"

Reply:
Addressed in 27ae1ba318f9521275e70681914ea37036850c8e.